SPEC.md

```diff
@@ -66,13 +66,13 @@ Each KSS documentation block consists of three parts: a description of what the
 
 ### The description section
 
-The description should be plain sentences of what the CSS rule or hierarchy does and looks like. A good description gives guidance toward the application of elements the CSS rules style.
+The description should be plain sentences of what the CSS rule or hierarchy does or looks like. A good description gives guidance toward the application of elements the CSS rules style.
```

Comment:
Earlier in the spec it says "Each KSS documentation block consists of three parts: a description of what the element does **or** looks like" (emphasis mine). This later wording should be the same.